docs/ramp_fitting/source/main.rst

```diff
@@ -37,7 +37,11 @@ a given pixel, the final slope is determined as a weighted average from all
 intervals and is written to a file as the primary output product.  In this
 output product, the 4-D GROUPDQ from all integrations is compressed into 2-D,
 which is then merged (using a bitwise OR) with the input 2-D PIXELDQ to create
-the output PIXELDQ.
+the output PIXELDQ.  If the ramp fitting step is run by itself, the output file 
+name will have the suffix '_RampFit'; if the ramp fitting step is run as the 
```

Comment:
One slight wrinkle here is that if one runs a step like so:

`strun jwst.rampfit.RampFitStep myfile_uncal.fits`

the output will have the `_RampFitStep` suffix.  If instead you run it using an input config file:

`strun rampfit.cfg myfile_uncal.fits`

it will have a `_RampFit` suffix, i.e. a suffix based off whatever the `name` keyword is set to in the .cfg file.  For other pipeline steps, many of our default config files have inconsistent naming. Some are camel case, some are not.  Might be a good idea to decide on a standard and stick with it.  Maybe lower case?

All this is controlled in step.py in the `__init__()` and `from_config_file()` methods.